documentation/providers/loopia.md

```diff
@@ -0,0 +1,158 @@
+  }
+}
+```
+{% endcode %}
+
+### Variables
+
+* `username` - string - your @loopiaapi created username
+* `password` - string - your loopia API password
+* `debug` - string - Set to true for extra debug output. Remove or set to false to prevent extra debug output. 
+* `rate_limit_per` - string - See Rate Limiting below.
+
+There is no test endpoint. Fly free, grasshopper.
+
+Turning on debug will show the XML requests and responses, and include the
+username and password from your `creds.json` file. If you want to share these, 
+like for a Github issue, be sure to redact those from the XML.
+
+
+### Rate Limiting
+
+Loopia rate limits requests to 60 per minute.
+
+From their [web-site](https://www.loopia.com/api/rate_limiting/):
+```text
+You can make up to 60 calls per minute to LoopiaAPI. Of those, a maximum of 15 can be domain searches.
+```
+
+Depending on how many requests you make, you may encounter a limit. Modification
+of each DNS record requires at least one API call. 🤦
+
+Example: If the rate is 60/min and you make two requests every second, the 31st
+request will be rejected. You will then have to wait for 29 seconds, until the
+first request’s age reaches one minute. At that time, it will be dropped from
+the calculation, and you can make another request. One second later, and
+generally every time an old request’s age falls out of the sliding window
+counting interval, you can make another request.
+
+Your per minute quota is 60 requests and in your settings you
+ specified `Minute`. DNSControl will perform at most one request per second.
+ DNSControl will emit a warning in case it breaches the quota.
+
+The setting `rate_limit_per` controls this behavior and accepts
+ a case-insensitive value of
+- `Hour`
+- `Minute`
+- `Second`
+
+The default for `rate_limit_per` is `Second`.
+
+In your `creds.json` for all `LOOPIA` provider entries:
+
+{% code title="creds.json" %}
+```json
+{
+  "loopia": {
+    "TYPE": "LOOPIA",
+    "username": "your-loopia-api-account-id@loopiaapi",
+    "password": "your-loopia-api-account-password",
+    "debug": "true", // Set to true for extra debug output. Remove or set to false to prevent extra debug output. 
+    "rate_limit_per": "Minute"
+  }
+}
+```
+{% endcode %}
+
+## Usage
+
+Here's an example DNS Configuration `dnsconfig.js` using the provider module.
+Even though it shows how you use Loopia as Domain Registrar AND DNS Provider,
+you're not forced to do that (thank god).
+
+
+{% code title="dnsconfig.js" %}
+```javascript
+// Providers:
+var REG_LOOPIA = NewRegistrar("loopia");
+var DSP_LOOPIA = NewDnsProvider("loopia");
+
+// Set Default TTL for all RR to reflect our Backend API Default
+// If you use additional DNS Providers, configure a default TTL
+// per domain using the domain modifier DefaultTTL instead.
+// also check this issue for [NAMESERVER TTL](https://github.com/StackExchange/dnscontrol/issues/176).
```

Comment:
There's no Markdown possible in a code block.

```suggestion
// also check this issue for NAMESERVER TTL: https://github.com/StackExchange/dnscontrol/issues/176.
```

Reply:
Removed this.